go.mod

```diff
@@ -1,17 +1,11 @@
 module github.com/matrix-org/rageshake
 
+go 1.16
```

Comment:
the readme claims "rageshake requires Go version 1.11 or later."

I have no particular desire to support ancient Go versions, but we better update the documentation and the CI if we want to update it.

Reply:
Updated the readme and go.mod to say 1.15 and made matrix-org/pipelines#171 to update the CI.